tests/test_deposit_nulls.nim

```diff
@@ -0,0 +1,60 @@
+import os
+import macros
+import std/json
+import pkg/unittest2
+import stint
+import ../web3
+import ../web3/[conversions, ethtypes]
+
+proc `==`(x, y: Address): bool {.borrow, noSideEffect.}
+proc `==`(x, y: Quantity): bool {.borrow, noSideEffect.}
+
+suite "Deposit contract":
+  test "deposits with nully values":
+    for jsonExample in parseFile(getAppDir() & "/test_deposits.json"):
+      discard $jsonExample
+
+  test "passing nully values to normal convertors":
+    var resAddress: Address
+    var resFixedBytes: FixedBytes[5]
+    var resQuantity: Quantity
+    var resRlpEncodedBytes: RlpEncodedBytes
+    var resTypedTransaction: TypedTransaction
+    var resUInt256: UInt256
+    var resUInt256Ref: ref UInt256
+
+    # TODO: @tavurth
```

Comment:
@zah as I understand it `DynamicBytes` is more a type and cannot be instanced as a `var`?